clusterloader2/testing/list/deployment.yaml

```diff
@@ -27,10 +26,14 @@ spec:
         args:
         - --namespace={{.Namespace}}
         - --uri={{.Uri}}
+        - --inflight={{.inflight}}
+        - --qps={{.qps}}
+        - --content-type={{.contentType}}
```

Comment:
This flag is not available in the existing image. We first need merge the changes to `main.go` and push the image. Could you split the PR so we first update the image?